src/drc/api/tests/test_enkelvoudiginformatieobject.py

```diff
@@ -471,11 +471,29 @@ def test_eio_delete(self):
         lock = self.client.post(f"{eio_url}/lock").data["lock"]
         self.client.patch(eio_url, {"beschrijving": "beschrijving2", "lock": lock})
 
+        url_unlock = get_operation_url(
+            "enkelvoudiginformatieobject_unlock", uuid=eio.uuid
+        )
+        unlock = self.client.post(url_unlock, {"lock": lock})
+
         response = self.client.delete(eio_url)
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
         self.assertFalse(EnkelvoudigInformatieObjectCanonical.objects.exists())
         self.assertFalse(EnkelvoudigInformatieObject.objects.exists())
 
+    def test_eio_delete_fails_on_locked(self):
+        eio = EnkelvoudigInformatieObjectFactory.create(beschrijving="beschrijving1")
+
+        eio_url = reverse(
+            "enkelvoudiginformatieobject-detail", kwargs={"uuid": eio.uuid}
+        )
+        lock = self.client.post(f"{eio_url}/lock").data["lock"]
+
+        response = self.client.delete(eio_url)
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertNotEqual(lock, "")
```

Comment:
ik snap deze assert niet. lock is een response dat kan nooit een lege string zijn? Wil je niet kijken of je een error krijgt die zegt dat deze resource locked is.

Reply:
`Locked objects cannot be destroyed`

Die dus neem ik aan